exporter/streams.go

```diff
@@ -4,22 +4,26 @@ import (
 	"github.com/gomodule/redigo/redis"
 	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
+	"strconv"
+	"strings"
 )
 
 // All fields of the streamInfo struct must be exported
 // because of redis.ScanStruct (reflect) limitations
 type streamInfo struct {
-	Length           int64 `redis:"length"`
-	RadixTreeKeys    int64 `redis:"radix-tree-keys"`
-	RadixTreeNodes   int64 `redis:"radix-tree-nodes"`
-	Groups           int64 `redis:"groups"`
+	Length           int64  `redis:"length"`
+	RadixTreeKeys    int64  `redis:"radix-tree-keys"`
+	RadixTreeNodes   int64  `redis:"radix-tree-nodes"`
+	LastGeneratedId  string `redis:"last-generated-id"`
+	Groups           int64  `redis:"groups"`
 	StreamGroupsInfo []streamGroupsInfo
 }
 
 type streamGroupsInfo struct {
 	Name                     string `redis:"name"`
 	Consumers                int64  `redis:"consumers"`
 	Pending                  int64  `redis:"pending"`
+	LasDeliveredtId          string `redis:"last-delivered-id"`
```

Comment:
`LasDeliveredtId` --> `LastDeliveredId` ?

Reply:
Right. changed to `parseStreamItemID` in my last commit.